core/unit_test/sycl/TestSYCL_InterOp_Graph.cpp

```diff
@@ -80,6 +80,15 @@ TEST(TEST_CATEGORY, graph_instantiate_and_debug_dot_print) {
 
   ASSERT_EQ(graph.native_graph().get_nodes().size(), 2u);
 
+#if defined(_GLIBCXX_RELEASE) && _GLIBCXX_RELEASE < 9
+  GTEST_SKIP()
+      << "The GNU C++ Library (libstdc++) versions less than 9.1 "
+         "require linking with `-lstdc++fs` when using std::filesystem";
```

Comment:
```suggestion
         "require linking with `-lstdc++fs`";
```
Please keep messages aligned with what we did elsewhere.

Reply:
This is a better comment in my opinion. I'm happy to change it for the CUDA and HIP test as well if you prefer.

Reply:
Yes, please.